indexer/feature_meta.cpp

```diff
@@ -125,6 +128,26 @@ bool Metadata::TypeFromString(string_view k, Metadata::EType & outType)
   return true;
 }
 
+void Metadata::ClearPOIAttribs()
+{
+  for (auto i = m_metadata.begin(); i != m_metadata.end();)
+  {
+    if (i->first != Metadata::FMD_ELE &&
```

Comment:
Should this list be manually updated after adding other metadata fields?..

Reply:
Well, not obvious, but I didn't have a better solution.